app/views/content/what-is-a-pgce.md

```diff
@@ -18,12 +19,28 @@ keywords:
   - State-school
 ---
 
-A PGCE is a teaching qualification. It stands for ‘postgraduate certificate in education’. 
+PGCEs and PGDEs are teaching qualifications. They stand for ‘postgraduate certificate in education’, and 'postgraduate diploma in education'.
 
-The course you take to get a PGCE is a mixture of learning education theory, and being placed in a school classroom to learn from experience.
+## Getting your PGCE
 
-Another qualification is a PGDE, which stands for ‘postgraduate diploma in education’. A PGDE course is very similar to a PGCE, the difference being the amount of academic credits you are awarded at the end of your course. 
+The course you take to get a PGCE is a mixture of learning education theory, and school placements where you will be put in a classroom to learn from experience.
 
-If you want to teach in a state school in England, you will also need to get ‘qualified teacher status’ (QTS) alongside your PGCE or PGDE course.
+Entry requirements for a PGCE course usually include an undergraduate degree or an equivalent qualification. The course will usually take 1 year to complete full time, or 2 years part time.
+
+[Find a postgraduate teacher training course](https://www.gov.uk/find-postgraduate-teacher-training-courses).
+
+### Types of PGCE courses
+
+You can do a PGCE course at a university, or through a school-led teacher training programme.
+
+You can do a PGCE in primary education, secondary education, or further/adult education.
+
+## Postgraduate diploma in education (PGDE)
```

Comment:
Is there a particular reason why this is heading is different from the one before

ie it isn't 'Getting a PGDE'

Seems slightly inconsistent to have one written about in full and the other with a verb at the front?

But not a major comment if this is deliberate for SEO!

Reply:
wasn't deliberate for SEO - it was more because the first section talks about the PGCE course, what you need to get onto it, where to find courses, whereas the PGDE section is just explaining what a PGDE is, less about how to 'get' your PGDE like the first section. But see your point about consistency, might just change both headings to 'PGCE' and 'PGDE'?